pywbem/_subscription_manager.py

```diff
@@ -1190,14 +1200,27 @@ def _create_destination(self, server_id, dest_url, owned):
                         dest_inst = server.conn.GetInstance(dest_path)
                         self._owned_destinations[server_id][i] = dest_inst
                     return dest_inst
-            dest_path = server.conn.CreateInstance(dest_inst)
+            dest_path = server.conn.CreateInstance(dest_inst,
+                                                   namespace=server.interop_ns)
             dest_inst = server.conn.GetInstance(dest_path)
             self._owned_destinations[server_id].append(dest_inst)
         else:
             # Responsibility to ensure it does not exist yet is with the user
             dest_path = server.conn.CreateInstance(dest_inst)
             dest_inst = server.conn.GetInstance(dest_path)
 
+        # If systemname does not yet exist, create it
+        if server_id not in self._systemnames:
+            self._systemnames[server_id] = dest_inst['SystemName']
+        else:
+            if self._systemnames[server_id] != dest_inst['SystemName']:
+                warnings.warn(
+                    _format("WBEM server may be returning different SystemName "
+                            "property for different operations: received  "
+                            "{0!A} and {0!A}. Owned destinations may not be "
+                            "found.", self._systemnames[server_id],
+                            dest_inst['SystemName']),
+                    UserWarning)
```

Comment:
If we remove the SystemName from the detection of owned instances as proposed in another comment, the detection of owned instances is not affected by different SystemName values anymore. In fact, it does not matter that much whether they are different, so I think we can remove the entire else path here.

Reply:
DONE